content/knowledge-base/guides/self-signed-ca-certs.md

```diff
@@ -11,7 +11,7 @@ registries, it must be configured to trust the CA and Intermediate certs.
 
 Crossplane needs to be installed via the Helm chart with the 
 `registryCaBundleConfig.name` and `registryCaBundleConfig.key` parameters 
-defined. See [Install Crossplane]({{<ref "../getting-started/install-configure" >}}).
+defined. See [Install Crossplane]({{<ref "../../v1.10/getting-started/install-configure" >}}).
```

Comment:
Should this be linking to a particular version>

Reply:
Updated with fb195a2a23209e4b1b971ea4cb40441ceb7d9c30